tests/lib/tools/tests.cleanup

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh -e
+
+show_help() {
+    echo "usage: tests.cleanup prepare"
+    echo "       tests.cleanup defer <cmd> [args]"
+    echo "       tests.cleanup restore"
+}
+
+cmd_prepare() {
```

Comment:
It could be avoided if it is called on first defer, or call it on prepare-each, so in the test we dont need to prepare/restore

Reply:
All three commands operate on a file present in the current working directory. I don't know if spread's prepare-each operates in that location.

Separately, as with restore, I like the explicit nature of this operation. It allows one to sequence other code either before or after the restore stack.